src/uu/chown/src/chown.rs

```diff
@@ -7,13 +7,18 @@
 
 // spell-checker:ignore (ToDO) COMFOLLOW Chowner Passwd RFILE RFILE's derefer dgid duid
 
+// Clippy bug: https://github.com/rust-lang/rust-clippy/issues/7422
+#![allow(clippy::nonstandard_macro_braces)]
+
```

Comment:
Can you try removing this? The issue seems to have been fixed, maybe we don't even need this anymore :)

Reply:
Yeah removed